docs/usage.md

```diff
@@ -3,130 +3,441 @@
-
-FortiAI operates in two distinct modes &ndash; Playbook Blocks and Conversation Mode &ndash; to provide intelligent assistance and guidance to security professionals: 
-
-- The **Playbook Blocks** mode is active in the playbook designer and provides users with two options
-    - Playbook Generation
-    - Response Plans / Jinja / Playbook How To's
-
-- The Conversation mode is active in the list view and detail view of modules in FortiSOAR and provides users with the **Ask a question** option. 
-
-<table>
-    <tr>
-        <th>EXAMPLE</th>
-    </tr>
-    <tr>
-        <td>In the Conversation mode:
-            <ul>
-                <li>click the 'FortiAI' icon on the lower-right corner in the alert detail view.
-                    <p><img src="./res/ai_bot_icon.png" alt="Advisor bot icon in FortiSOAR"></p>
-                </li>
-                <li>Click the <strong>FortiAI</strong> to open the flyout to contextually communicate with the FortiAI
-                    <p><img src="./res/ai_bot_dialog.png" alt="Advisor dialog"></p>
-                </li>
-                <li>Click the <strong>Clear Conversation</strong> button to manually clear a conversation in FortiAI The <em>Clear Conversation</em> button clears the conversations only from the current Conversation mode and not from <strong>Playbook Generation Mode</strong>.</li>
-            </ul>
-        </td>
-    </tr>
-</table>
-
-<table>
-    <th>NOTE</th>
-    <td>When a user logs out, all conversations are automatically cleared.</td>
-</table>
-
-## Use cases
-
-### Build response plans using FortiAI
+The **FortiAI** solution pack offers a variety of functions through its **AI Assistant** to managing and automating tasks related to cybersecurity workflows. Following is an overview of the supported functions:
 
-Consider a situation where you are assigned to create a plan of action for investigating a ransomware event.
+- **Generate Playbook**: Generate, create a playbook or workflow, or playbook steps.
+- **Navigate Pages**: Navigate, open, or move to a specific page or module.
+- **Link Record**: Link or correlate a record with other relevant records. For example: correlating an indicator to an existing alert or incident.
+- **Create Record**: Create a record for the specified type, page, or module.
+- **Update Record**: Update a record using the updated information provided.
+- **Block Indicator**: Block an indicator using a configured connector.
+- **Enrich Indicator**: Fetch, get, or retrieve the latest reputation of an indicator.
+- **Escalate To Incident**: Escalate, elevate, or promote the alert to an incident.
+- **Setup War Room**: Establish, set up, initiate, or create a war room for an incident.
 
-You can use *FortiAI* to help with this task by opening a playbook in the playbook designer and clicking the **Response Plans / Jinja / Playbook How To's** option in the FortiAI flyout.
+>[!NOTE]
```

Comment:
Move this note to below where we are actually explaining create, update, link usecases

Reply:
Done